README.md

```diff
@@ -1,3 +1,21 @@
 # holoviz_tasks
 
 Shared GHA workflows and tasks used to maintain the HoloViz repositories
+
+## Release
+
+To make an releasetag the commit with the version number and push it to the repo.
+
+```bash
+git tag -a v0.1.X -m "Version 0.1.X"
+git push --tags v0.1.X
+```
+
+Afterwards merge the commits up tag into `v0` branch. This is branch is used across
```

Comment:
I had a look at other Github actions (setup-python, checkout) and they use a tag instead of a branch for v0, v1, etc.

I assume they do something along these lines (not tested):

```
git tag -a v0.1.X -m "Version 0.1.X"
git tag -a --force v0 -m "Version 0"
git push origin v0.1.X v0 --force
```

Reply:
I tried that, but I felt it was insincere to have a tag that keeps changing, whereas it is expected for a branch.

Reply:
Ok fair enough.